.gitignore

```diff
@@ -8,3 +8,4 @@ Gemfile.lock
 .ruby-*
 tags
 config/*.yml
+extra/end-to-end-test.sh
```

Comment:
Not that I mind that much, but this is very specific :)

Reply:
Yeah fixed.